src/components/Content/Pursuance/views/DiscussView.js

```diff
@@ -12,24 +12,23 @@ const leapChatUrl = "http://localhost:8080/#GiddinessPuttRegisterKioskLucidityJo
 class DiscussView extends Component {
 
   componentWillMount(){
-    const { match: { params: { pursuanceId } }, tasks, getTasks } = this.props;
-    if (!tasks.taskMap['1_1']) {
+    const { match: { params: { pursuanceId, taskGid } }, tasks, getTasks } = this.props;
+    if (!tasks.taskMap[taskGid]) {
       getTasks(pursuanceId);
     }
   }
 
   render() {
-    const { pursuances, tasks } = this.props;
-    // TODO: Un-hardcode after demo
-    const taskGid = '1_1';
+    const { pursuances, tasks, match: { params: { taskGid } } } = this.props;
     const task = tasks.taskMap[taskGid];
-    if (!task) {
-      return <div>Ain't nobody got task fo' that.</div>
+    if(!task){
+      return <div className="no-task">Ain't nobody got task fo' that.</div>
     }
     const assignedPursuanceId = task.assigned_to_pursuance_id;
+    const subtaskGids = task.subtask_gids;
     return (
       <div className="discuss-ctn">
-        <iframe className="leapchat-frame" title="Leapchat" src={leapChatUrl} />
+        <iframe className="leapchat-frame" title="Leapchat" src={leapChatUrl + taskGid} />
```

Comment:
@Moe-Shoman Great! Could you also please make it so clicking on each `Discuss Task` icon will take the user to that particular chat room?

Reply:
@elimisteve Yeah, is that not working correctly?